package.json

```diff
@@ -49,6 +49,7 @@
     "watch": "grunt",
     "install-and-build": "npm install && npm run build",
     "test": "mocha test",
+    "ui_test": "cd ./docker && docker build -t insight_local . && cd .. && ./node_modules/protractor/bin/webdriver-manager update && (./node_modules/protractor/bin/webdriver-manager start > webdriver.log 2>&1 &) && sleep 30 && ./node_modules/protractor/bin/protractor ./test/ui/conf.js",
```

Comment:
We are using prefixes usually, like `test:ui` and please add to `test` script which supposed to run all tests, like `mocha test && npm run test:ui`.

Reply:
You can ommit `./node_modules/protractor/bin/`

Reply:
Quite strange to have docker build here. Let's move it to Travis build.

Reply:
modified